frontend/src/app/fireworks/fireworks/fireworks.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-fireworks',
+  templateUrl: './fireworks.component.html',
+  styleUrls: ['./fireworks.component.scss']
+})
+export class FireworksComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
Can probably remove, along with the empty constructor, and the `implements OnInit` above